pl_bolts/callbacks/verification/batch_gradient.py

```diff
@@ -0,0 +1,199 @@
+from typing import Any, Callable, List, Optional
+
+import torch
```

Comment:
For type hints of `BatchGradientVerificationCallback`:
```suggestion
import torch
from pytorch_lightning import LightningModule, Trainer
```